### PR TITLE
Fix outdated copyright info, fix license not showing in the about dialog

### LINKED ISSUE #2508: in the about pane the copyright is 2004 - 2014
should be 2015 based on the release :) and is using (c) when the copyright symbol is part of unicode available on any platform anywhere ©

### PR DESCRIPTION
**Pull request by Umcaruje**
Fixes #2508 

Pics: 
screenshot from 2016-01-28 23 18 00
screenshot from 2016-01-28 23 18 19
screenshot from 2016-01-28 23 18 39

### CODE REVIEW

**Comment by zonkmachine** [comment]
The license turns up nicely.

File: src/gui/AboutDialog.cpp [hunk 1]
```diff
@@ -46,9 +46,12 @@ AboutDialog::AboutDialog(QWidget* parent) :
 					arg( QT_VERSION_STR ).
 					arg( GCC_VERSION ) );
 
+	copyrightLabel->setText( copyrightLabel->text().
+					arg( LMMS_PROJECT_COPYRIGHT ) );
+
 	authorLabel->setPlainText( embed::getText( "AUTHORS" ) );
 
-	licenseLabel->setPlainText( embed::getText( "COPYING" ) );
+	licenseLabel->setPlainText( embed::getText( "LICENSE.txt" ) );
```
Comment (tresf) [inline src/gui/AboutDialog.cpp]: Good catch.

File: src/gui/AboutDialog.cpp [hunk 2]
```diff
@@ -46,9 +46,12 @@ AboutDialog::AboutDialog(QWidget* parent) :
 					arg( QT_VERSION_STR ).
 					arg( GCC_VERSION ) );
 
+	copyrightLabel->setText( copyrightLabel->text().
+					arg( LMMS_PROJECT_COPYRIGHT ) );
+
```
Comment (tresf) [inline src/gui/AboutDialog.cpp]: Not a huge fan of this logic, `me.setValue(me.getValue() + " foo");` as multiple calls can look like `foo foo foo foo` but it's already used in the constructor and it's already used in the lines preceding, it's OK.  Offtopic, but we probably need to make a decision on if we even want these `.ui` files around as they just make the code hard to read IMO. :wink:

**Comment by tresf** [comment]
Commit looks good and @zonkmachine tested.  Merging.  :+1:

**Pull request merged**